price-feeder/oracle/provider/binance.go

```diff
@@ -237,6 +239,15 @@ func (p *BinanceProvider) messageReceived(messageType int, bz []byte) {
 	}
 	if len(tickerResp.LastPrice) != 0 {
 		p.setTickerPair(tickerResp)
+		telemetry.IncrCounter(
+			1,
+			"websocket",
+			"message",
+			"type",
+			"ticker",
+			"provider",
+			config.ProviderBinance,
```

Comment:
Just using the only constants that carry the names of the providers here - maybe we should have an additional set of constants in the `providers` package, but that seems a bit overkill for this

Reply:
Would also like @RafilxTenfen 's input on any other metrics that might be useful